arch/arm/core/swap_helper.S

```diff
@@ -22,7 +22,7 @@ _ASM_FILE_PROLOGUE
 
 GTEXT(__svc)
 GTEXT(__pendsv)
-GTEXT(_do_kernel_oops)
+GTEXT(z_do_kernel_oops)
 GTEXT(_arm_do_syscall)
```

Comment:
Why _arm_do_syscall is not updated?

Reply:
It's one of those cases that exist only in .S files. The starting point for these changes was the Coccinelle script, and Coccinelle only detects code in .h and .c files. Doing a manual fixup for this.